packages/app/src/runner/event-manager.ts

```diff
@@ -363,6 +363,20 @@ export class EventManager {
     this.ws.emit('watch:test:file', config.spec)
 
     if (config.isTextTerminal || config.experimentalInteractiveRunEvents) {
+      this.localBus.on('run:end', async () => {
```

Comment:
FYI-This will need to account for promises and leads to same issues I said that was left where we can still see issues switching specs for the open-mode implementation to be marked as resolved.

Also to correctly send the mocha results / the flavor that matches the documented API, we will need to update what we emit with 'run:end'